docker/fedora/keylime_rust.Dockerfile

```diff
@@ -62,3 +64,11 @@ WORKDIR ${KEYLIME_HOME}
 RUN git clone https://github.com/keylime/keylime.git && \
 cd keylime && \
 cp keylime.conf /etc/keylime.conf
```

Comment:
Could you please remove this? It is not necessary anymore as we install the agent configuration in the `make install` step.